pkg/controller/pod_driver.go

```diff
@@ -81,13 +81,14 @@ func (p *PodDriver) SetMountInfo(mit mountInfoTable) {
 }
 
 func (p *PodDriver) Run(ctx context.Context, current *corev1.Pod) error {
+	podStatus := getPodStatus(current)
+	klog.V(6).Infof("[PodDriver] start handle pod %s/%s, status: %s", current.Name, current.Namespace, podStatus)
```

Comment:
stick to print output convention?

```suggestion
	klog.V(6).Infof("[PodDriver] start handle pod %s/%s, status: %s", current.Namespace, current.Name,  podStatus)
```